GoogleForCreators/web-stories-wp
# Text Presets: Fix default values

## Summary

Improves default values for text presets, particularly font size, line height and font weight.

## Relevant Technical Choices

Cleans up text presets a little bit to add stricter prop type checking and help separate preset data and element data.

## To-do



## User-facing changes

* Changes minimum font size for text presets to 1.2em / roughly 16px.  
  <img width="275" alt="Screenshot 2020-06-26 at 14 43 21" src="https://user-images.githubusercontent.com/841956/85858770-79989980-b7bc-11ea-9c65-0f0d127823c1.png">
* Fixes a bug where the subheading preset did have an invalid font weight of 600, whereas [Roboto](https://fonts.google.com/specimen/Roboto) only supports 500 or 700, nothing in-between
  <img width="263" alt="Screenshot 2020-06-26 at 14 45 46" src="https://user-images.githubusercontent.com/841956/85858867-9af98580-b7bc-11ea-8dd2-541aa8d16136.png">


## Testing Instructions

1. Insert new text elements from the text presets, including the "quick action" via the ➕ icon in the tab.
1. Verify that "body text" and the element added via "quick action" show a font size of 16
1. Verify that all presets have a line  height of 1.5 when added.
1. Verify  that the "subheading" element shows "Medium" for the font weight instead of "Multiple"

---



Fixes #2665
Fixes #2041
